src/modules/commander/Commander.cpp

```diff
@@ -1894,7 +1898,10 @@ Commander::set_home_position()
 		const vehicle_local_position_s &lpos = _local_position_sub.get();
 		_heading_reset_counter = lpos.heading_reset_counter; // TODO: should not be here
 
-		fillLocalHomePos(home, lpos);
+		vehicle_attitude_s att{};
+		_vehicle_attitude_sub.copy(&att);
+
+		fillLocalHomePos(home, lpos, att);
```

Comment:
So this is good. My assumption is that you always set local position and you may also set global position. That's why we only bill the attitude in the local position check right? (since GPS can't tell you your attitude anyway).

Reply:
I don't understand what you mean.

This is always set based on local position. Local position is initialized / referenced to a global position, if possible.

This just adds the attitude at that moment in.

Reply:
@julianoes I have no idea what I meant; so long ago. Just tidying open issues - so consider this a reminder that something was happening here :-)